### PR TITLE
docs: Pod examples

### PR DESCRIPTION
**Pull request by nhlfr**
A little instruction with examples to how run a VM with
virtlet.

---
This change is [Reviewable](https://reviewable.io/reviews/mirantis/virtlet/20)

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# virtlet pod examples
+
+To be able to use the following pod definitions, please fetch the images and serve
+them via HTTP server. This can be done like this:
+
+```sh
+wget http://download.cirros-cloud.net/0.3.4/cirros-0.3.4-x86_64-disk.img -O cirros
+wget http://ftp.ps.pl/pub/Linux/fedora-linux/releases/24/CloudImages/x86_64/images/Fedora-Cloud-Base-24-1.2.x86_64.qcow2 -O fedora
+sudo twistd -n web --path . -p 80
```
Comment (jellonek) [inline examples/README.md]: This requires twisted. Please change this to `python2 -m SimpleHTTPServer 80 .`

File: examples/virt-cirros.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: virtlet-example-cirros
+spec:
+  containers:
+    - name: cirros
+      image: 172.19.0.1/cirros
```
Comment (jellonek) [inline examples/virt-cirros.yaml]: IMO Docs should have an info where this ip comes from and how this should be set.

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+```
+
+Then you will be able to create this pods in your local k8s cluster:
+
+```sh
+./cluster/kubectl.sh $GOPATH/src/github.com/Mirantis/virtlet/examples/virt-cirros.yaml
```
Comment (jellonek) [inline examples/README.md]: Looks like this misses `create -f` part.

File: examples/README.md (hunk 1 above)
Comment (nhlfr) [inline examples/README.md]: done

File: examples/virt-cirros.yaml (hunk 2 above)
Comment (nhlfr) [inline examples/virt-cirros.yaml]: done

File: examples/README.md (hunk 3 above)
Comment (nhlfr) [inline examples/README.md]: done

**Comment by jellonek** [comment]
Small nits - `python2` and please change `ip r` to `ip route` (we should not use aliases in examples).

**Comment by nhlfr** [comment]
done

**Comment by pigmej** [comment]
:lgtm:

---

Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/20#-:-KTrWpg6GbkgxAfxdSz0:bnfp4nl)*

**Pull request merged**